### PR TITLE
MOSIP-13118 Create separate library for Machine, Center, Device and OSI validation

### PR DESCRIPTION
**Pull request by neosoftSatish**

### CODE REVIEW

File: sandbox/registration-processor-camel-routes-lost-mz.xml [hunk 1]
```diff
@@ -105,10 +105,98 @@
...
+                <simple>${bodyAs(String)} contains '"internalError":true'</simple>
+                <to uri="eventbus:retry-bus-in" />
+            </when>
+            <otherwise>
+                <to uri="eventbus:error-bus-in" />
+            </otherwise>
+        </choice>
+    </route>
+	<route id="introducer-validator-->demo-dedupe new route">
+        <from uri="eventbus:introducer-validator-new-bus-out" />
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-camel-routes-lost-mz.xml]: this should be last bus out. Also osi to demo do we have a route in old one ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-camel-routes-lost-mz.xml (hunk 1 above)
Comment (neosoftSatish) [reply sandbox/registration-processor-camel-routes-lost-mz.xml]: removed introducer to demo-dedupe and used lost bus out...

Comment (neosoftSatish) [reply sandbox/registration-processor-camel-routes-lost-mz.xml]: removed introducer to demo-dedupe and used lost bus out...

Comment (neosoftSatish) [reply sandbox/registration-processor-camel-routes-lost-mz.xml]: removed introducer to demo-dedupe and used lost bus out...

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**